cypress/e2e/with_mock_data/catalogueItems.cy.ts

```diff
@@ -324,15 +323,35 @@ describe('Catalogue Items', () => {
           'Please enter a valid Drawing link. Only "http://" and "https://" links with typical top-level domain are accepted.'
         ).should('exist');
         cy.findByRole('button', { name: 'Next' }).should('be.disabled');
+
+        // details - negative number input validation test
+        cy.findByLabelText('Cost (£) *').clear();
```

Comment:
```suggestion
        // details - negative number input validation test
        cy.findByLabelText('Drawing link').clear();
        cy.findByLabelText('Cost (£) *').clear();
```
I would clear the drawing link field from the previous test, just so that you're certain the negative number validation is causing the Next button to be disabled

Reply:
good spot :)